user_dir = "/opt/gateone/users"
log_file_num_backups = 10
certificate = "/opt/gateone/certificate.pem"
sso_service = "HTTP"
dtach = True
embedded = False
session_logging = False
sso_realm = "EXAMPLE.COM"
auth = None
log_file_prefix = "/var/log/gateone/webserver.log"
log_file_max_size = 104857600
session_dir = "/tmp/gateone-log"
cookie_secret = "A45CharacterStringGeneratedAtRandom012345678"
syslog_session_logging = False
syslog_facility = "daemon"
address = "0.0.0.0"
debug = False
command = "/opt/gateone/plugins/ssh/scripts/ssh_connect.py -S '/tmp/gateone/%SESSION%/%r@%h:%p' -a '-oUserKnownHostsFile=%USERDIR%/%USER%/known_hosts'"
session_timeout = "5d"
keyfile = "/opt/gateone/keyfile.pem"
port = 443
